hugogonum/content/home/about.md

```diff
@@ -0,0 +1,58 @@
++++
+# About/Biography widget.
+
+date = "2016-04-20T00:00:00"
+draft = false
+
+widget = "about"
+
+# Order that this section will appear in.
+weight = 1
+
+# List your academic interests.
+# [interests]
+#  interests = [
+#    "Artificial Intelligence",
+#    "Computational Linguistics",
+#    "Information Retrieval"
+#  ]
+
+# List your qualifications (such as academic degrees).
+# [[education.courses]]
+#  course = "PhD in Artificial Intelligence"
+#  institution = "Stanford University"
+#  year = 2012
+
+#[[education.courses]]
+#  course = "MEng in Artificial Intelligence"
+#  institution = "Massachusetts Institute of Technology"
+#  year = 2009
+
+#[[education.courses]]
+#  course = "BSc in Artificial Intelligence"
+#  institution = "Massachusetts Institute of Technology"
+#  year = 2008
+ 
++++
+
+Gonum is a set of packages designed to make writing numeric and scientific
+algorithms productive, performant, and scalable.
+
+Gonum contains libraries for [matrices and linear algebra](https://godoc.org/gonum.org/v1/gonum/mat);
+[statistics](https://godoc.org/gonum.org/v1/gonum/stat), 
+[probability](https://godoc.org/gonum.org/v1/gonum/stat/distuv) 
+[distributions](https://godoc.org/gonum.org/v1/gonum/stat/distmv), 
+and [sampling](https://godoc.org/gonum.org/v1/gonum/stat/sampleuv); tools for
+[function differentiation](https://godoc.org/gonum.org/v1/gonum/diff/fd), 
+[integration](https://godoc.org/gonum.org/v1/gonum/integrate/quad),
+and [optimization](https://godoc.org/gonum.org/v1/gonum/optimize);
+[network](https://godoc.org/gonum.org/v1/gonum/graph) creation and analysis; and more.
+
+We encourage you to [get started](http://gonum.org/post/introtogonum/) with Go and Gonum if
+
+* You are tired of sluggish performance, and with fighting vectorization and C
```

Comment:
Suggest

```
* You are tired of sluggish performance, and fighting C and vectorization.
* You are struggling with managing programs as they grow larger.
* You struggle to re-use -- even the code you tried to make reusable.
* You would like easy access to parallel computing.
* You want code to be fully transparent, and want the ability to read the source code you use.
* You'd like a compiler to catch mistakes early, but hate fighting linker and unintelligible compile errors.
```

probably also in the article. This makes the lines generally shorter and fixes odd line breaks.